java/jdbc-relational-duality-example/src/main/java/org/oracle/BooksManager.java

```diff
@@ -0,0 +1,330 @@
+/* Copyright (c) 2021, 2022, Oracle and/or its affiliates.
+This software is dual-licensed to you under the Universal Permissive License
+(UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+either license.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+DESCRIPTION
+BooksManager - Class that has requests methods for Books and Book Copy tables and their duality view
+*/
+
+package org.oracle;
+
+import oracle.jdbc.OracleType;
+import oracle.sql.json.OracleJsonObject;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class BooksManager {
+
+    public static void retrieveBooks(DatabaseConfig pds) {
```

Comment:
The code looks fine but could you please use 2 space characters for indentation? This is to be consistent with the rest of the code samples.